tests/standalone_activity_test.go

```diff
@@ -709,8 +709,69 @@ func (s *standaloneActivityTestSuite) TestCompletedActivity_CannotTerminate() {
 	require.Error(t, err)
 }
 
+func (s *standaloneActivityTestSuite) Test_ScheduleToCloseTimeout_WithRetry() {
```

Comment:
This test could be flaky when CI is under load.
2 seconds is fairly short to ensure at least one attempt is issued.